src/test/java/com/practise/tdd/CheckoutShould.java

```diff
@@ -29,4 +29,14 @@ public void return_100_for_cart_having_AA() {
 		assertEquals(new Integer(100), checkout.total("AA"));
 	}
 	
+	@Test
+	public void return_130_for_cart_having_AAB() {
+		assertEquals(new Integer(130), checkout.total("AAB"));
+	}
+	
+	@Test
+	public void return_130_for_cart_having_AAA() {
+		assertEquals(new Integer(130), checkout.total("AAA"));
```

Comment:
This seemed like next logical step, as I had already validated 3 combinations before any eligible discount.